steps/twitter-message/0.0.3/step.yml

```diff
@@ -0,0 +1,78 @@
+title: twitter-message
+summary: |
+  Send message on twitter
+description: |
+  This step is used to send message on twitter.
+website: https://github.com/kawaiseb/bitrise-step-twitter-message
+source_code_url: https://github.com/kawaiseb/bitrise-step-twitter-message
+support_url: https://github.com/kawaiseb/bitrise-step-twitter-message/issues
+published_at: 2018-02-25T23:09:45.334133397+01:00
+source:
+  git: https://github.com/kawaiseb/bitrise-step-twitter-message.git
+  commit: aae5b27f3d6313bed4ebff6a9121cd086cea207e
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_required: true
+    summary: Your application's Consumer Key and Secret are used to authenticate requests
+      to the Twitter Platform.
+    title: Consumer Key (API Key)
+  tw_consumer_key: null
+- opts:
+    is_required: true
+    title: Consumer Secret (API Secret)
+  tw_consumer_secret: null
+- opts:
+    is_required: true
+    title: Access Token Secret
```

Comment:
May it would be great to describe how to generate/get the twitter Access Token Secret.